pkgs/development/haskell-modules/with-packages-wrapper.nix

```diff
@@ -4,6 +4,7 @@
 # platform that has NCG support
 , useLLVM ? false
 , withHoogle ? false
+, installDocumentation ? true
```

Comment:
Could you add a comment on this option?  Maybe just the same text from the release-notes file?

Reply:
Done.